schema/vrs.yaml

```diff
@@ -413,35 +434,81 @@ definitions:
       gene_id:
         $ref: "#/definitions/CURIE"
 
-  CytobandInterval:
+  # - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
+  # Quantity
+
+  Quantity:
```

Comment:
I'm fine with this for Quantity.

Reply:
Just to be clear, removing Quantity from the terms and model doc would put _CopyCount_ at the same level as _Feature_, _Location_, and _Primitive Concepts_. Do we want to keep in .rst but drop in yaml only?

Reply:
I think we should leave this an allow for comments during review. I think we are at a point that we need to discuss how we consistently deal with simple numeric values vs bounds. I want to move quickly, but I do think we should take a moment to discuss and learn from existing work in this area and determine if any of that should inform our approach. I will be able to get more into it if we leave this in for now so that I can post the comments for review.

The nutshell argument is that we should be able to treat simple things simply, which I could argue is not what we do know with forcing the need for simple positions or copycounts to be a min/max range. We are trying to use a complex set of contsraints to satisfy a bunch of use cases and i believe we can do better. But let's leave that for review to decide.